invisible_cities/cities/detsim_simulate_electrons_test.py

```diff
@@ -0,0 +1,137 @@
+import os
+import numpy  as np
+import tables as tb
+import pandas as pd
+
+from collections import namedtuple
+from operator    import itemgetter
+
+from pytest import fixture
+
+import invisible_cities.core.system_of_units as units
+
+from invisible_cities.io.mcinfo_io import load_mchits_df
+
+from invisible_cities.cities.detsim_simulate_electrons import generate_ionization_electrons
+from invisible_cities.cities.detsim_simulate_electrons import drift_electrons
+from invisible_cities.cities.detsim_simulate_electrons import diffuse_electrons
+
+@fixture(params = ["Kr83_nexus_v5_03_00_ACTIVE_7bar_3evts.MCRD.h5"])
+def nexusMC_filename(request, ICDATADIR):
+    return os.path.join(ICDATADIR, request.param)
+
+
+@fixture
+def MChits_and_detsim_params(nexusMC_filename):
+
+    #hits
+    hits_df    = load_mchits_df(nexusMC_filename)
+    event_ids  = hits_df.index.levels[0].values
+    #choose random event
+    evt = np.random.choice(event_ids)
+    hits = hits_df.loc[evt]
+
+    xs = hits["x"].values
+    ys = hits["y"].values
+    zs = hits["z"].values
+    energies = hits["energy"].values
+
+    # electron simulation parameters
+    wi = 22.4 * units.eV
+    fano_factor = 0.15
+    drift_velocity = 1.00 * units.mm / units.mus
+    lifetime       = 7.00 * units.ms
+
+    transverse_diffusion   = 1.00 * units.mm / units.cm**0.5
+    longitudinal_diffusion = 0.30 * units.mm / units.cm**0.5
+
+
+    typename = "MChits_and_detsim_params"
+    field_name = """xs ys zs energies wi fano_factor lifetime drift_velocity
+                    transverse_diffusion longitudinal_diffusion"""
+
+    nt = namedtuple(typename, field_name)
```

Comment:
Why not use keyword arguments in `namedtuple`?

```suggestion
    nt = namedtuple(typename = "MChits_and_detsim_params", 
                    field_names = """xs ys zs energies wi fano_factor lifetime
                       drift_velocity transverse_diffusion longitudinal_diffusion""")
```
You may prefer some other layout of `field_names`.